src/main/java/com/github/manolo8/darkbot/core/manager/GuiManager.java

```diff
@@ -239,4 +261,61 @@ public boolean canTickModule() {
         return main.hero.locationInfo.isLoaded();
     }
 
+    private final RectangleImpl gameScreenRect = new RectangleImpl();
+
+    @Override
+    public Area.Rectangle getViewBounds() {
+        gameScreenRect.set(0, 0, MapManager.clientWidth, MapManager.clientHeight);
```

Comment:
??
a rectangle that always starts at 0,0 makes no sense whatsoever. The view bounds should start on the top-left corner of your visible area, usually your location sits in the middle of this rectangle